src/main/java/com/contrastsecurity/maven/plugin/ContrastInstallAgentMojo.java

```diff
@@ -21,41 +21,94 @@
 import org.apache.maven.plugins.annotations.Parameter;
 import org.apache.maven.project.MavenProject;
 
+/**
+ * Includes the Contrast Java agent in integration testing to provide Contrast Assess runtime
+ * security analysis
```

Comment:
I can’t remember if it’s standard practice, but a lot of the JavaDocs are missing punctuation at the end. Some the JavaDoc does have punctuation at the end, so it seems inconsistent at least.

Reply:
I typically don't end single sentence docs with punctuation because there's no following sentence to distinguish a barrier between. I again looked up what the standard practice is. The [JavaDoc Style Guide](https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#styleguide) does not speak to this directly, but the [JavaDocStyle CheckStyle rule](https://checkstyle.sourceforge.io/config_javadoc.html#JavadocStyle) says this

> Ensures the first sentence ends with proper punctuation (That is a period, question mark, or exclamation mark, by default). Javadoc automatically places the first sentence in the method summary table and index. Without proper punctuation the Javadoc may be malformed. All items eligible for the {@inheritDoc} tag are exempt from this requirement.

That's good enough for me. I'll add punctuation to my JavaDoc from now on

Reply:
This being the language that insists on terminating statements with a semi-colon, I should have known 😆